### PR TITLE
Dynamic titles

### LINKED ISSUE #202: Add dynamic titles to pages
On the old site, page titles change for different pages: https://thecourseforum.com/reviews/new

Can implement this with a title block in index that's replaced in each template

### PR DESCRIPTION
**Pull request by jackliu612**
# Status
Done, In progress, etc

# What I did

Added dynamic titles to many pages

## Issues addressed

Closes #202 

## Screenshots

image

### CODE REVIEW

**Review by Bartleby2718 — CHANGES_REQUESTED** [review]
1. I think we're missing `course_professor.html` and `user_reviews.html`.
2. It seems like `theCourseForm` is cut off most of the time. Should we change it to `tCF`?
3. `theCourseForum` is repeated in all blocks, but I guess it's more readable to do so, so no worries. Just wanted to point that out.

**Review by alex-shen1 — COMMENTED** [review]
Honestly a little on the fence about this — I like it for stuff like the About page, but I'm not sure if it makes sense to have stuff like entire course names/longer departments in the title of a page. The main point of the title is to display the name of the website, which gets completely pushed to the side if the name is long enough.

**Comment by alex-shen1** [comment]
One potential compromise solution would be to set everything to `Browse - theCourseForum` instead of having individual department/course/instructor names.

File: tcf_website/templates/search/search.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends "base/base.html" %}
 {% load static %}
 
+{% block title %}{{displayed_query}} - theCourseForum{% endblock %}
```
[the remark this answers is not in the document]
Comment (jackliu612) [reply tcf_website/templates/search/search.html:4]: image
Google is like this so it's probably fine how it is

File: tcf_website/templates/course/course.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {% extends "base/base.html" %}
 {% load static %}
 
+{% block title %}{{course.title}} - theCourseForum{% endblock %}
```
[the remark this answers is not in the document]
Comment (jackliu612) [reply tcf_website/templates/course/course.html:4]: This and Browse is just following what is on tcf1 as an example. I can change or keep it how it is

**Review by Bartleby2718 — APPROVED** [review]

**Pull request merged**